ui/app/templates/components/file-to-array-buffer.hbs

```diff
@@ -6,7 +6,7 @@
       {{/if}}
     </label>
     <div class="file has-name is-fullwidth">
-      <label class="file-label">
+      <div class="file-label">
         <input class="file-input" type="file" onchange={{action "pickedFile"}} data-test-file-input />
```

Comment:
This input no longer has a label. I feel like some a11y checks should have caught that 🤔 We should at least add a screenreader-only label here